submissions/LuckyDnepr/js-dom/aside-menu-action.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+import { readJSON } from "./readJSON.js";
+
+const dataBaseUrl = "./data-films.json";
+
+export async function loadContentByID(id, container) {
+  const data = await readJSON(dataBaseUrl);
+  container.innerHTML = "";
+  const film = data.find((film) => film.ID === +id);
+  container.appendChild(makeTitle(film));
```

Comment:
That's a lot of reaching out to DOM just to switch content on one action. Consider wrapping all these elements with a [document fragment](https://developer.mozilla.org/ru/docs/Web/API/Document/createDocumentFragment) and using only one `appendChild`.